TicketSwap/solar
# Align icons with design using Figma as single source of truth

*BREAKING CHANGE*: Renaming CopyLink icon to Link icon

# Description

Added missing icons, renamed CopyLink to Link, updated Icon story

## Changes

Added:

- `AlertRounded`
- `Barcode`
- `Compass`
- `EventType`
- `Pause`
- `QrCode`
- `Shuffle`
- `Snooze`
- `StarHalf`
- `Watch`

Renamed:

- `CopyLink` to `Link`


## How to test

- Checkout this branch
- `$ yarn && yarn storybook`
- Go to the Iconography and check if all icons are there.
